apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -1062,8 +1062,18 @@ def open(self):
         self._target.handle_semihost_io()
         self._target._stdout = BlockingStream()
         self._target._stdin = BlockingStream()
-        self._model.run(blocking=False, timeout=100)
-        self._wait_for_semihost_init()
+        for i in range(3):
+            try:
+                _LOG.info("Iteration %d", i)
+                self._model.run(blocking=False, timeout=100)
+                self._wait_for_semihost_init()
+                break
+            except self._iris_lib.error.IrisError:
+                continue
+            else:
```

Comment:
note: indent 1 less level for this block

Reply:
Oh, thanks for the note!